src/coreComponents/physicsSolvers/multiphysics/SinglePhaseReservoirAndWells.hpp

```diff
@@ -26,13 +26,13 @@
 namespace geos
 {
 
-template< typename SINGLEPHASE_RESERVOIR_SOLVER >
-class SinglePhaseReservoirAndWells : public CoupledReservoirAndWellsBase< SINGLEPHASE_RESERVOIR_SOLVER,
+template< typename RESERVOIR_SOLVER >
```

Comment:
Please add a `@tparam` that emphasize that we need a single phase solver (or the constraint is not valid anymore)?

Reply:
added, it still has to be single phase yes